.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "6"
+  - "12"
```

Comment:
Bumped Node version as was hitting an error with object spread operator being a problem and v6 is way old 🤷‍♂

Reply:
We should continue validating the library can run on Node 6, otherwise this is a breaking change in a sense. I'm okay with validating across multiple node versions. That being said, can you point me to the error you were seeing using the spread operator?

Reply:
@ctumolosus Here's an example of the build failure ... https://travis-ci.org/flowcommerce/redux-fetch/builds/613448331